### ISSUE TITLE
Flameshot zooms in & can't handle multiple screens on mac m1.

### ISSUE THREAD

**Issue by nohehf** [flameshot-org/flameshot#2612]
### Flameshot Version

Flameshot v11.0.0 (ad1bf28)
Compiled with Qt 5.15.2
darwin: 21.1.0
osx: 12.0

### Installation Type

Linux, MacOS, or Windows Package manager (apt, pacman, eopkg, choco, brew, ...)

### Operating System type and version

macOS Monterey v12.0.1, Apple M1 Pro

### Description

When trying to take a screenshot, flameshot insanely zooms in, leaving only the top-right part of the screen accessible to screen shot. I'm on google chrome, and have multiple screens. Also I can only screen shot the main screen.
 

### Steps to reproduce

1. Start flameshot screenshot onto google chrome

### Screenshots or screen recordings

https://user-images.githubusercontent.com/64989428/169247415-e2fffc09-8233-47d2-93a6-627cde732efc.mov

### System Information

<img width="635" alt="Screenshot 2022-05-19 at 10 19 47" src="https://user-images.githubusercontent.com/64989428/169247322-1780520f-575a-4dd4-becc-14059e119245.png">
<img width="447" alt="Screenshot 2022-05-19 at 10 20 26" src="https://user-images.githubusercontent.com/64989428/169247329-13766410-bd08-4918-8cc2-9ffa36d9979d.png">

**Comment by nohehf** [comment]
You're right, thought I had already done it my bad. Tho I still have always a little zoom in, which is pretty annoying (see video, how the graph at the bottom gets cropped).

https://user-images.githubusercontent.com/64989428/169377997-37609191-9750-42ba-9c2a-531ef478fbb0.mov

**Comment by mmahmoudian** [comment]
@nohehf  As far as I can see, this is not a zoom effect, but there is downwards shift of the image. Note the additional black region on top of the image. How does you monitor setup look like when you face this issue?

**Comment by nohehf** [comment]
Right, it's shifting, and I'm only on my laptop screen.

**Comment by borgmanJeremy** [comment]
The other issue specifically pointed out that this is on a screen with a notch. I wonder if its related to that and Qt is using the region below the notch to draw full screen instead of the region above the notch. I have no way to test / work on this since I have an air.

**Comment by jasonycw** [comment]
I have been experiencing the exact same issue on the macbook with notch. I think it's due to the way MacOS handling the notch with a "full screen application"

With normal screen, all full screen application will occupied the whole screen, so when Flameshot capture the screen and open a new full screen application for cropping, the screen looks fine.

However, on the macbook pro's notched screen, all full screen application cannot occupied the whole screen, so when when Flameshot capture the screen and open a new full screen application for cropping, the screen looks like it's shifted downward.

https://user-images.githubusercontent.com/4518597/173267761-d7220fe1-5351-4f20-ac3e-d1a4108d2d17.mov

https://user-images.githubusercontent.com/4518597/173267485-25fcb3c5-0387-4348-b343-bba52fe3597f.mov

You can try mimic the behavior on a regular screen by trigger Flameshot screenshot and without dragging anything, swipe touchpad to open mission control, drag Flamshot into a regular space, resize the Flameshot window. You will see the screen will not be resized and it will be "cropped"

I think if Flameshot can scale down the screenshot into within the windows size, then it should work better on a notched screen.

**Comment by jasonycw** [comment]
FYI, wIth the new version 12, the magnifier when taking screenshot also shift down/up a few pixel

**Comment by jasonycw** [comment]
Here is a screen capture for reference


https://user-images.githubusercontent.com/4518597/175845709-2b9c7b9c-cf42-4cd4-800a-752921e3ddda.mov

**Comment by jasonycw** [comment]
Here is a screen capture for reference

https://user-images.githubusercontent.com/4518597/175847391-e2235ac3-48f1-45e1-b21e-70f4e7392854.mov

**Comment by AmandaBSobrinho** [comment]
I was able to reproduce the same issue here in a MacBook without the notch by enabling the option "Automatically hide and show the menu bar in full screen", in Dock & Menu Bar (macOS 12.6).

**Comment by monochroumme** [comment]
Happens in my case too, it's because of the notch on 2021 Macbooks. Apps don't take the top of the screen to not be hidden under the notch and the same happens in the flameshot app. In other apps, the scroll area is increased to compensate the decrease of the height of the window whereas flameshot simply crops the bottom of the screen. Any way to resolve the issue?
![telegram-cloud-photo-size-2-5195078186977839243-y](https://user-images.githubusercontent.com/18418245/204103551-d9cd3e9e-7743-4ed4-94ec-3569b1423825.jpg)
![image](https://user-images.githubusercontent.com/18418245/204103744-288b2511-ce22-4337-8436-eb0491f7d628.png)

**Comment by jorpilo** [comment]
Same here, is there any way to prevent it from shifting the screen?

**Comment by sonterix** [comment]
The same issue on Mac OS. Is there any way to fix it or I should search for another app? Because as I can see devs are ignoring this from May 19, 2022. 

_Flameshot v12.1.0 (96c2c82e)
Compiled with Qt 5.15.5
darwin: 21.6.0
osx: 12.6_

**Comment by kYem** [comment]
For me this seems even more extreme, but maybe that to do with some animation?
![image](https://github.com/flameshot-org/flameshot/assets/3390897/103fc995-60bf-43c1-9aa7-6190ee9db3f4)

vs build in macOS screenshot works just fine without any adjustment

![image](https://github.com/flameshot-org/flameshot/assets/3390897/d671ad07-6893-42aa-9b4b-6df0c65a73d8)

**Comment by volk-ov** [comment]
+1 
Flameshot v12.1.0 (96c2c82e)
Compiled with Qt 5.15.5
MacOS 14.3.1 (23D60)

**Comment by julian-zatloukal** [comment]
+1
Flameshot v12.1.0 (96c2c82e)
Compiled with Qt 5.15.5
osx: 14.4

**Comment by davidneyravyr** [comment]
+1
Flameshot v12.1.0 (https://github.com/flameshot-org/flameshot/commit/96c2c82e62bdabad6c20b97004c493f69f90a986)
Compiled with Qt 5.15.5
osx: 14.4

please FIX!

**Comment by kopach** [comment]
Restarting Flameshot as mentioned here https://github.com/flameshot-org/flameshot/issues/2612#issuecomment-1131678178 works for me. Kinda workaround, but at least I need to do this one after monitor connect/disconnect

**Comment by davidneyravyr** [comment]
@kopach remember this thread is for "notch" inside macbook screen. is mentioned by https://github.com/flameshot-org/flameshot/issues/2612#issuecomment-1328093681

**Comment by Untersander** [comment]
I have the same issue with the image shifting downwards on a macbook pro without a notch.

**Comment by NilsTessier** [comment]
a workaround is to enable the app to be displayed under the notch like suggested [here](https://apple.stackexchange.com/questions/442217/is-there-a-way-to-allow-the-apps-to-be-displayed-behind-the-notch-when-a-menubar)  :
1. Find the app in Finder
2. Press ⌘+i
3. Check Scale to fit below built-in camera
